tests/integration_tests/data_steward/cdr_cleaner/cleaning_rules/rdr_observation_source_concept_id_suppression_test.py

```diff
@@ -79,17 +79,17 @@ def setUp(self):
         now_prediabetes = 43530333
         prefer_not_to_ans = 903079
 
-        load_statements = []
+        self.load_statements = []
         # create the string(s) to load the data
         for tmpl in self.insert_fake_participants_tmpls:
             query = tmpl.render(fq_table_name=self.fq_table_names[0],
                                 age_prediabetes=age_prediabetes,
                                 meds_prediabetes=meds_prediabetes,
                                 now_prediabetes=now_prediabetes,
                                 prefer_not_to_ans=prefer_not_to_ans)
-            load_statements.append(query)
+            self.load_statements.append(query)
 
-        self.load_test_data(load_statements)
```

Comment:
Close.  Because the tables are created in the individual test setup function now, it doesn't exist at this point.  The table isn't created until after `super().setUp()` is called.  So I can't load data into the table because it doesn't exist at this point.  I could have called
```
super().setUp()
self.load_test_data(load_statements)
```
I instead moved the load from the test setup to the first part of the test definition.  This is the same sequence as the code segment listed here, but achieved slightly differently.

Reply:
Okay, gotcha!